sdk/peering/Azure.ResourceManager.Peering/tests/Scenario/PeeringTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Peering.Models;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Peering.Tests
+{
+    internal class PeeringTests : PeeringManagementTestBase
+    {
+        private ResourceGroupResource _resourceGroup;
+        private PeeringCollection _peeringCollection => _resourceGroup.GetPeerings();
+
+        public PeeringTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [SetUp]
+        public async Task TestSetUp()
+        {
+            _resourceGroup = await CreateResourceGroup();
+        }
+
+        [RecordedTest]
+        [Ignore("ASN validation can take up to 3 business days after submitting PeerASN request.")]
```

Comment:
As we ignored this test, do we still need the corresponding test record? I only found very few http records in the record file, maybe we can remove these files.